### PR TITLE
Node update

### PR DESCRIPTION
**Pull request by loneil**
# Description
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2290

Update node and packages.
Normalize package.json config structures (move to files) to match the way we do it in other apps.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

Did not update morgan/npmlog, ticket to remove those coming up https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2351

### CODE REVIEW

File: openshift/app.bc.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ objects:
         dockerStrategy:
           from:
             kind: DockerImage
-            name: docker.io/node:lts-alpine
+            name: docker.io/node:16.13.0-alpine
```
Comment (jujaga) [inline openshift/app.bc.yaml:58]: 16.13.1

**Review by jujaga — APPROVED** [review]

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://dgrsc-dev.apps.silver.devops.gov.bc.ca/pr-76

File: openshift/app.bc.yaml (hunk 1 above)
Comment (loneil) [reply openshift/app.bc.yaml:58]: thx 👍

**Comment by loneil** [comment]
Not sure why `test-app (10.x)` and `test-frontend (10.x)` are hanging around in the checks... PR was opened without 10.x being in there at all

**Comment by bcgov-nr-csst** [comment]
> Not sure why `test-app (10.x)` and `test-frontend (10.x)` are hanging around in the checks... PR was opened without 10.x being in there at all

Adjusted checks to not need 10.x ones (settings > branches > edit > Protect matching branches)

**Review by TimCsaky — APPROVED** [review]

**Review by MichaelChung123 — APPROVED** [review]

**Pull request merged**